deployments/kubernetes/chart/forecastle/templates/deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
       labels:
 {{ include "forecastle.labels.selector" . | indent 8 }}
     spec:
+      nodeSelector: {{- toYaml .Values.forecastle.deployment.nodeSelector | nindent 8 }}
```

Comment:
Its better if we make nodeSelector optional. Otherwise this will become an issue for people who don't need a nodeSelector. And a good way to do it is to just add it with either `with` or an normal if check will also work

Reply:
I believe it works anyway - but anyway `with` is cleaner so I've changed it.